tests/lib/rules/display-name.js

```diff
@@ -548,5 +561,16 @@ ruleTester.run('display-name', rule, {
     errors: [{
       message: 'Component definition is missing display name'
     }]
+  }, {
+    code: [
+      'import React, { createElement } from "react";',
```

Comment:
Let's also add cases for `var { createElement } = React;` and `var createElement = React.createElement;`.

Probably too difficult to cover `var c = React.createElement;` tho.